transcom/mymove
# B 18644 actual move date not future backend

## B-18633

## Summary
When updating the actual move date for a PPM the date should not be any date in the future.

These are to be checked by a reviewer.


### How to test

1. Create a PPM move
2. Login as a SC, edit and submit the move
3. As customer click Upload PPM documents
4. Enter Departure Date(it is required, this is the same as actual move date)
5. Verify seeing "UpdateError" when attempting to set date in the future

## Screenshots
<img width="1151" alt="Screenshot 2024-03-18 at 10 40 44 AM" src="https://github.com/transcom/mymove/assets/146969726/59fb6386-5c9e-4a05-96c7-185892b5aaec">
